### PR TITLE
[TL][#121231237] Add credo and example .credo.exs

### PR DESCRIPTION
**Pull request by Teapane**
[Story](https://www.pivotaltracker.com/story/show/121231237)

### CODE REVIEW

File: elixir/.credo.exs [hunk 1]
```diff
@@ -0,0 +1,108 @@
+This file contains the configuration for Credo and you are probably reading
```
Comment (martinstreicher) [inline elixir/.credo.exs]: Is this line missing a `#`?

Comment (martinstreicher) [inline elixir/.credo.exs]: Is this line missing a `#`?

Comment (Teapane) [inline elixir/.credo.exs]: Yep, amended. 😄

**Comment by martinstreicher** [comment]
Cool so far.

**Comment by colinrymer** [comment]
:+1: LGTM :shipit:

**Pull request merged**